engines/python/setup/djl_python/huggingface.py

```diff
@@ -308,12 +313,10 @@ def inference(self, inputs):
                     idx += 1
 
                 formatter = parameters[i].get("output_formatter")
-                if formatter == "json":
-                    outputs.add_property(f"batch_{i}_Content-Type",
-                                         "application/json")
-                elif formatter == "jsonlines":
-                    outputs.add_property(f"batch_{i}_Content-Type",
-                                         "application/jsonlines")
+                if formatter in OUTPUT_FORMATTER_TO_CONTENT_TYPE:
+                    outputs.add_property(
+                        f"batch_{i}_Content-Type",
+                        OUTPUT_FORMATTER_TO_CONTENT_TYPE[formatter])
```

Comment:
The formatter might not be a str type, better use a function

Reply:
Done